site/_data/demos/navigation.yml

```diff
@@ -17,9 +17,58 @@
         path: /react/grid/demos/featured/chart-integration
 /react/chart:
   - title: DEMOS
```

Comment:
Maybe remove this section? Because it looks badly
![screen shot 2018-09-07 at 18 12 28](https://user-images.githubusercontent.com/15093892/45228969-4aeebd80-b2cd-11e8-9af9-c52ab85fb4af.png)

Reply:
Yes, we can remove 'Demos' title